libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1369,6 +1369,10 @@ uint16_t AP_GPS::gps_yaw_cdeg(uint8_t instance) const
 
 void AP_GPS::send_mavlink_gps_raw(mavlink_channel_t chan)
 {
+    if (params[0].type == GPS_TYPE_NONE || drivers[0] == nullptr) {
```

Comment:
We continue to send this message to "clear" the GCS's concept of what the GPS data is.  So it might stop displaying a position on a map or something.

That's my recollection, anyway.

OK, I looked it up: https://github.com/ArduPilot/ardupilot/commit/e995a198bd71f7b5c03f1a6784be749525cc4428

This behaviour is on purpose.


I think this might be reasonable:
```suggestion
    if (params[0].type == GPS_TYPE_NONE) {
```

Reply:
I agree with the comment.